--- CMakeLists.txt	2015-01-19 21:47:41.000000000 +0100
+++ CMakeLists.txt	2016-05-19 03:07:31.584590364 +0200
@@ -34,6 +34,7 @@
 ELSEIF(APPLE)
     SET(APP_TARGET Mudlet)
 ELSE()
+    INCLUDE_DIRECTORIES("/usr/include/lua-5.1")
     SET(APP_TARGET mudlet)
 ENDIF()
 
@@ -54,3 +55,10 @@
 # i.e. ./src/src.pro
 
 ADD_SUBDIRECTORY(src)
+
+INSTALL(FILES src/mudlet DESTINATION bin
+    PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+                GROUP_READ             GROUP_EXECUTE
+                WORLD_READ             WORLD_EXECUTE)
+INSTALL(DIRECTORY src/mudlet-lua DESTINATION share/mudlet
+    FILES_MATCHING PATTERN "*.lua")
