--- src/TLuaInterpreter.cpp	2017-05-03 22:24:02.000000000 +0200
+++ src/TLuaInterpreter2.cpp	2017-05-04 12:22:46.350175033 +0200
@@ -13398,15 +13398,9 @@
 
 void TLuaInterpreter::loadGlobal()
 {
-#if defined(Q_OS_MAC)
-    // Load relatively to MacOS inside Resources when we're in a .app bundle,
-    // as mudlet-lua always gets copied in by the build script into the bundle
-    QString path = QCoreApplication::applicationDirPath() + "/../Resources/mudlet-lua/lua/LuaGlobal.lua";
-#else
     // Additional "../src/" allows location of lua code when object code is in a
     // directory alongside src directory as occurs using Qt Creator "Shadow Builds"
-    QString path = "../src/mudlet-lua/lua/LuaGlobal.lua";
-#endif
+    QString path = "/usr/share/mudlet/mudlet-lua/lua/LuaGlobal.lua";
 
     int error = luaL_dofile( pGlobalLua, path.toLatin1().data() );
     if( error != 0 )
