diff --git a/src/mudlet-lua/lua/LuaGlobal.lua b/src/mudlet-lua/lua/LuaGlobal.lua
index 2ca6bf7..1913c9f 100644
--- a/src/mudlet-lua/lua/LuaGlobal.lua
+++ b/src/mudlet-lua/lua/LuaGlobal.lua
@@ -148,7 +148,7 @@ local prefixes = { "./../../src/mudlet-lua/lua/",
                    "./../Resources/mudlet-lua/lua/",
                    "./mudlet-lua/lua/",
                    "./lua/",
-                   "mudlet.app/Contents/Resources/mudlet-lua/lua/" }
+                   "/usr/share/mudlet/mudlet-lua/lua/"}
 
 -- add default search paths coming from the C++ side as well
 if getMudletLuaDefaultPaths then
