diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3ae56a2..6843054 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -59,6 +59,7 @@ if(WIN32)
 elseif(APPLE)
   set(APP_TARGET Mudlet)
 else()
+  INCLUDE_DIRECTORIES("/usr/include/lua-5.1")
   set(APP_TARGET mudlet)
 endif()
 
@@ -273,3 +274,10 @@ endif(CCACHE_FOUND)
 add_subdirectory(src)
 add_subdirectory(3rdparty/communi)
 add_subdirectory(translations/translated)
+
+INSTALL(FILES src/mudlet DESTINATION bin
+  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+              GROUP_READ             GROUP_EXECUTE
+              WORLD_READ             WORLD_EXECUTE)
+INSTALL(DIRECTORY src/mudlet-lua DESTINATION share/mudlet
+  FILES_MATCHING PATTERN "*.lua")
