diff --git a/CMakeLists.txt b/CMakeLists.txt
index c322f9b..dbe1beb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -43,6 +43,7 @@ if(WIN32)
 elseif(APPLE)
   set(APP_TARGET Mudlet)
 else()
+  INCLUDE_DIRECTORIES("/usr/include/lua-5.1")
   set(APP_TARGET mudlet)
 endif()
 
@@ -230,3 +231,10 @@ endif(CCACHE_FOUND)
 add_subdirectory(src)
 add_subdirectory(3rdparty/communi)
 add_subdirectory(3rdparty/lua_yajl)
+
+INSTALL(FILES src/mudlet DESTINATION bin
+  PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+              GROUP_READ             GROUP_EXECUTE
+              WORLD_READ             WORLD_EXECUTE)
+INSTALL(DIRECTORY src/mudlet-lua DESTINATION share/mudlet
+  FILES_MATCHING PATTERN "*.lua")
