diff --git a/cmake/SelectPcsx2Plugins.cmake b/cmake/SelectPcsx2Plugins.cmake
index c0e395d..4ffe214 100644
--- a/cmake/SelectPcsx2Plugins.cmake
+++ b/cmake/SelectPcsx2Plugins.cmake
@@ -99,14 +99,7 @@ endif()
 #---------------------------------------
 #			cdvdGigaherz
 #---------------------------------------
-if(NOT EXISTS "${CMAKE_SOURCE_DIR}/plugins/cdvdGigaherz" OR NOT Linux)
-    set(cdvdGigaherz FALSE)
-elseif(Linux AND GTKn_FOUND AND LIBUDEV_FOUND)
-    set(cdvdGigaherz TRUE)
-else()
-    set(cdvdGigaherz FALSE)
-    print_dep("Skip build of cdvdGigaherz: miss some dependencies" "${msg_dep_cdvdgiga}")
-endif()
+set(cdvdGigaherz FALSE)
 #---------------------------------------
 
 #---------------------------------------
