diff --git a/cmake/SearchForStuff.cmake b/cmake/SearchForStuff.cmake
index 3bf2f21..00d93bd 100644
--- a/cmake/SearchForStuff.cmake
+++ b/cmake/SearchForStuff.cmake
@@ -192,8 +192,6 @@ include_directories(${CMAKE_SOURCE_DIR}/common/include
 #----------------------------------------
 include(ApiValidation)
 
-WX_vs_SDL()
-
 # Blacklist bad GCC
 if(GCC_VERSION VERSION_EQUAL "7.0" OR GCC_VERSION VERSION_EQUAL "7.1")
     GCC7_BUG()
